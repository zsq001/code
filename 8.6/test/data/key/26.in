001110001000010100100011001101110100001001101010100011011100111100101001101111001101111011011000011100011010011101111000010111100011011101111001111110101010001001000111001001111010110110111111101010000000100110001110110000001100110
