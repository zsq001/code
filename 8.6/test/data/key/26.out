38
2 8 10 22 32 34 38 41 52 58 62 76 85 95 100 103 109 115 119 125 130 139 145 149 158 161 163 167 170 173 176 188 191 199 202 212 220 224
